plugins/kotlin-dataframe/README.md

```diff
@@ -0,0 +1,10 @@
+## :plugins:kotlin-dataframe
+
+This module, published as "compiler-plugin-all", holds the Kotlin 2.x Compiler plugin of DataFrame.
+
+A [work-in-progress](https://github.com/Kotlin/dataframe/issues/704)
+plugin for your Gradle project that can generate on-the-fly column accessors for the compiler and IDE even without
```

Comment:
Not sure about "Gradle project" detail. It will work in notebooks too in not-so-distant future

Reply:
fair, it might even work in maven, won't it?